### PR TITLE
VM monitoring adapt the latest version `0.50.0`of the `opentelemetry-collector`

### PR DESCRIPTION
**Pull request by wankai123**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [X] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/changelog/docs/en/changes/changes.md).

### CODE REVIEW

File: oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/oc/OCMetricHandler.java [hunk 1]
```diff
@@ -54,14 +55,15 @@
 
 @Slf4j
 public class OCMetricHandler extends MetricsServiceGrpc.MetricsServiceImplBase implements Handler {
-
+    public static final String HOST_NAME_LABEL = "node_identifier_host_name";
```
Comment (wu-sheng) [inline oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/oc/OCMetricHandler.java:58]:
```suggestion
    private static final String HOST_NAME_LABEL = "node_identifier_host_name";
```

File: docs/en/changes/changes.md [hunk 2]
```diff
@@ -45,6 +45,7 @@
 * Support autocomplete tags in logs query.
 * Enhance Deprecated MetricQuery(v1) getValues querying to asynchronous concurrency query
 * Fix the pod match error when the service has multiple selector in kubernetes environment.
+* VM monitoring adapt the latest version of the `opentelemetry-collector`.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:48]:
```suggestion
* VM monitoring adapts the 0.50.0 of the `opentelemetry-collector`.
```

**Review by kezhenxu94 — APPROVED** [review]
If tests passed

**Review by soulsoul — COMMENTED** [review]
it's ok to me

**Comment by chenxiaohu** [comment]
@wankai123 deploy.docker FEATURE_FLAGS=single-node,vm can not receive any data into oap but no error message.

**Pull request merged**